src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Update/UpdateDialogDto.cs

```diff
@@ -61,6 +61,12 @@ public sealed class UpdateDialogDto
     /// <example>2022-12-31T23:59:59Z</example>
     public DateTimeOffset? ExpiresAt { get; set; }
 
+    /// <summary>
+    /// Indicates if this dialog is intended for API consumption only and should not be displayed in user interfaces.
+    /// When true, the dialog will not be visible in portals designed for human users, but will remain accessible via API.
```

Comment:
Burde vi ha en kommentar her om Transmission content? 
(Og ev. ha samme info på Create sin summary? 👀 )
```suggestion
    /// Indicates if this dialog is intended for API consumption only and should not be displayed in user interfaces.
    /// When true, the dialog will not be visible in portals designed for human users, but will remain accessible via API.
    /// If any Transmissions were created without Content while this property was true, the flag cannot be reverted to false.
```